src/memory-analyzer/analyze_symbol.cpp

```diff
@@ -29,6 +31,66 @@ gdb_value_extractort::gdb_value_extractort(
 {
 }
 
+bool gdb_value_extractort::memory_scopet::contains(
+  const memory_addresst &point) const
+{
+  size_t begin_int = std::strtoul(begin.address_string.c_str(), NULL, 0);
```

Comment:
I wasn't sure if the `safe`-conversion would work on hex-numbers.

Reply:
They should, they use `stoll` which basically works the same way as the `str*` functions except it's easier to detect failures.

Reply:
+1

Reply:
Done and refactored so that the conversion is computed once in case of `distance`. See: https://github.com/diffblue/cbmc/pull/4578/commits/3812c28519a207a0e5f247f4b7fe2cd4db19af2b.